Libraries/StyleSheet/__tests__/processTransform-test.js

```diff
@@ -46,33 +56,51 @@ describe('processTransform', () => {
       expect(() =>
         processTransform([{translate: 10}]),
       ).toThrowErrorMatchingSnapshot();
+      expect(() =>
+        processTransform('translate(10)'),
```

Comment:
Once this is using a length value, it should be supported, as in CSS the second arg is optional and defaults to `0`, ie, it's the same as using `translateX`

Reply:
Fixed